File: passbook-management-api/src/main/java/com/uit/passbook_management_api/dto/request/MoSoRequest.java
package com.uit.passbook_management_api.dto.request;

import lombok.Data;

import java.math.BigDecimal;

@Data
public class MoSoRequest {
    private String tenKhachHang;
    private String cmnd;
    private String diaChi;
    private String loaiTietKiem;
    private BigDecimal soTienGuiBanDau;
}

File: frontend-csharp/Models/SoTietKiemModel/MoSoRequest.cs
using System.Text.Json.Serialization;

namespace frontend_csharp.Models.SoTietKiem.SoTietKiemModel
{
    public class MoSoRequest
    {
        [JsonPropertyName("tenKhachHang")]
        public string TenKhachHang { get; set; }

        [JsonPropertyName("cmnd")]
        public string Cmnd { get; set; }

        [JsonPropertyName("diaChi")]
        public string DiaChi { get; set; }

        [JsonPropertyName("loaiTietKiem")]
        public string LoaiTietKiem { get; set; }

        [JsonPropertyName("soTienGuiBanDau")]
        public decimal SoTienGuiBanDau { get; set; }
    }
}